README.md

```diff
@@ -1 +1,55 @@
-# terra-application
\ No newline at end of file
+<!-- Logo -->
+<p align="center">
+  <img height="128" width="128" src="https://github.com/cerner/terra-application/raw/master/terra.png" alt="terra logo" />
+</p>
+
+<!-- Name -->
+<h1 align="center">
+  Terra Application
+</h1>
+
+[![NPM version](https://badgen.net/npm/v/terra-application)](https://www.npmjs.org/package/terra-application)
+[![Cerner OSS](https://badgen.net/badge/Cerner/OSS/blue)](http://engineering.cerner.com/2014/01/cerner-and-open-source/)
+[![License](https://badgen.net/github/license/cerner/terra-application)](https://github.com/cerner/terra-application/blob/master/LICENSE)
+[![Build Status](https://badgen.net/travis/cerner/terra-application)](https://travis-ci.com/cerner/terra-application)
+[![Dependencies status](https://badgen.net/david/dep/cerner/terra-application)](https://david-dm.org/cerner/terra-application)
+[![devDependencies status](https://badgen.net/david/dev/cerner/terra-application)](https://david-dm.org/cerner/terra-application?type=dev)
+
+A framework to support application development with Terra components.
+
+* [Getting Started](#getting-started)
+* [Versioning](#versioning)
+* [Contributing](#contributing)
+* [LICENSE](#license)
+
+## Getting Started
+
+* Install with [npm](https://www.npmjs.com): `npm install --save terra-application`
+
+## Peer Dependencies
+
+terra-application requires no peer dependencies to be installed to properly function.
+
+## Versioning
+
+terra-application is considered to be stable and will follow [SemVer](https://semver.org/) for versioning.
+
+1. MAJOR versions represent breaking changes
+2. MINOR versions represent added functionality in a backwards-compatible manner
+3. PATCH versions represent backwards-compatible bug fixes
+
+Consult the component CHANGELOGs, related issues, and PRs for more information.
+
+## Contributing
```

Comment:
Should we add a section on testing? I assume there should maybe be tighter requirements for making larger/non-passive changes?

Reply:
The documentation in general could use some TLC. I'm going to synchronize the docs here with what I'm doing internally, but I won't have that ready for some time.